add port_quota demo tool to the getting started document (#587)
microsoft/ebpf-for-windows: docs/GettingStarted.md

If you're not already familiar with eBPF, or want a detailed walkthrough, see our [eBPF tutorial](tutorial.md).

For API documentation, see https://microsoft.github.io/ebpf-for-windows/
### Port leak and bind observability demo

> This section shows how to use eBPF for Windows in a demo that lets us control a UDP port leak by hooking an eBPF program at the bind layer of the windows networking stack via the EBPF_ATTACH_TYPE_BIND hook.

Reviewer: Not yet resolved since the term "bind layer" still appears without definition.  I'd suggest a different term.

> This section shows how to use eBPF for Windows in a demo that lets us control a UDP port leak by hooking an eBPF program at the bind layer of the Windows networking stack via the EBPF_ATTACH_TYPE_BIND hook.

Reviewer: ```suggestion
This section shows how to use eBPF for Windows in a demo that lets us control a UDP port leak by attaching an eBPF program to the socket bind() call via the EBPF_ATTACH_TYPE_BIND hook.
```

> This section shows how to use eBPF for Windows in a demo that lets us control a UDP port leak by hooking an eBPF program at the bind layer of the windows networking stack via the EBPF_ATTACH_TYPE_BIND hook.

Reviewer: See suggested change
